README.md

```diff
@@ -47,4 +47,8 @@ Contributions, issues, and feature requests are welcome!
 Give a ⭐️ if you like this project!
 
 ## Copyright
+<<<<<<< HEAD
```

Comment:
**Fix the information required in the ``README`` file:** 
- Add a "Acknowledgements" section that gives appropriate credit to Cindy Shin the author of the original design, as **required by the Creative Commons License**.
- Provide a ``License``.